bamboo_ci/ci_env.yaml

```diff
@@ -0,0 +1,14 @@
+dependencies:
+  - python=3.8
+  - pip
+  - numpy
+  - nltk
+  - cchardet
+  - crfpp==0.59
+  - skll==2.5
```

Comment:
Should this be skll>=2.5 too?

Reply:
No. From the PR description:

> Update ETS-internal CI bamoo build plan to use Python 3.8 and pinned SKLL to exactly v2.5 as that’s what we are still using at ETS.

Reply:
Right. I missed the commit description..